pages/install-device-page.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class InstallDevicePage {
+    readonly page: Page;
+    readonly secondDeviceItem: Locator;
+    readonly acceptButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.secondDeviceItem = page.getByLabel('Available devices').locator('svg').nth(2);
```

Comment:
could you please verify that Playwright inspector will give you this expression. Depending on your answer we might need to think how to name this variable or even if we should put here the exact item, because it might be more flexible to refer to the parent dom element and have a method to access the nth item.

Reply:
I got this from "codegen" tool, the svg is the unique element that is not variable. I also tried to craft something with locator "ul" instead but that did not work for some reason.

Reply:
Changed.